sbncode/LArRecoProducer/LArReco/TrajectoryMCSFitter.cxx

```diff
@@ -88,7 +106,10 @@ void TrajectoryMCSFitter::breakTrajInSegments(const recob::TrackTrajectory& traj
   }
   //then add last segment
   if (thislen>0.) {
+    auto endpointpos = traj.LocationAtPoint(nextValid);
+    bool endpointposgood = isInVolume(fiducialVolumes, endpointpos) && !isInVolume(excludeVolumes, endpointpos);
     breakpoints.push_back(traj.LastValidPoint()+1);
+    breakpointsgood.push_back(endpointposgood);
```

Comment:
Question: isn't `nextValid` here always an `recob::TrackTrajectory::InvalidIndex`?
The behaviour of `LocationAtPoint()` on an `InvalidIndex` is undefined, so we should make sure we are not triggering it.